bundles/org.openhab.binding.teleinfo/src/main/resources/OH-INF/thing/lsmmProdElectricityMeter.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="teleinfo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="lsmm_prod_electricitymeter" listed="false">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="serialcontroller"/>
+		</supported-bridge-type-refs>
+
+		<label>Linky Single-phase producer</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions